staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -77,11 +77,16 @@ type AuditWebhookOptions struct {
 	//
 	// Defaults to asynchronous batch events.
 	Mode string
+	// Configuration for batching webhook. Only used in batch mode.
+	BatchConfig *pluginwebhook.BatchBackendConfig
```

Comment:
What about using a non-pointer here? This is not optional, isn't it?

Reply:
SGTM, replaced with a value, no performance concerns anyway. Thanks!